### DISCUSSION TITLE
[QUESTION] Warp Image

### DISCUSSION THREAD

**Discussion by felipe-perin** [mono/SkiaSharp#1646] [Q&A]
How can I warping image with SkiaSharp, like this:

![warp_image](https://user-images.githubusercontent.com/15641314/109703926-cd8e2600-7b74-11eb-8393-d14eacc7a15b.jpg)

Anyone have idea about this?

**Comment by ziriax** [comment]
That should be possible with DrawVertices

https://docs.microsoft.com/en-us/dotnet/api/skiasharp.skcanvas.drawvertices?view=skiasharp-2.80.2

But how to do this exactly with a texture is unclear from the
documentation...

Actually many years ago I implemented such a moving least squares algorithm
on top of XNA; it is an amazing algorithm!

If you get it working, it would be nice if you could share it with us.

If not, ping me, I can help with that.

**Reply by felipe-perin** [reply]
thanks for reply.

I didn't work with XNA, but I heard about it very well.

I'm doing some tests with a converted Android code that I found on StackOverflow.
Unfortunately, I still haven't achieved the desired result.
Here is the code link:

https://stackoverflow.com/a/9239154

I will soon share the results.

Best regards.
